### PR TITLE
[DO NOT MERGE UNTIL 4098 IS CLOSED] Refactored .process CSS block. Moved to partial.

### PR DESCRIPTION
**Pull request by ghost**
## Blocked by #4098 

Closes department-of-veterans-affairs/vets.gov-team#657

- Moved process-list and list numbers to a partial, no-react CSS file.
- Changed `step` class to `process-step`.
- Changed one, two, etc. class names to list-*.
- Reduced selector specificity for .process ol, ul
- Added process--playbook class.
- Changed .process li.last to use :last-child.
- Added process--playbook, process-step--iterative clases.
- Renamed 'interactive' to more accurately-named 'badge--iterative'

### CODE REVIEW

**Comment by tiffanybbrown-va** [comment]
@melwoodard @rroueche Please review design and content changes.

@jcmeloni Feel free to add someone else for code review. I added you mostly for awareness.

**Comment by rroueche** [comment]
@tiffanybbrown-va awesome thanks! we have some changes to the pages in the Education section going live on Friday -- in this PR: https://github.com/department-of-veterans-affairs/vets-website/pull/4098

What's your input on how to handle? Seems like we have a couple options:

1. Update the `content/wip/education-plain-language-update` branch with these class changes now so that we can go ahead and merge this PR, and then when the education pages go live Friday they should be ok.

2. Wait to merge this PR until after the 'content/wip/education-plain-language-update' changes have gone live (Friday) and then update the education pages as a part of this branch to have the correct classes, then check in staging, then launch.

**Comment by tiffanybbrown-va** [comment]
@rroueche Ideally, we'd update the class names before #4098 was merged. _However_, I don't know how soon this PR will be merged. My fear is that this PR won't get merged before Friday and those will pages go out naked.

**Comment by rroueche** [comment]
@tiffanybbrown-va ok, in this situation then I'll request that we wait until #4098 is merged and then I'll come back to this one and update the classes for the Education pages in this branch. Is that ok w/everyone?

File: content/pages/disability-benefits/after-you-apply.md
Comment (melwoodard) [inline content/pages/disability-benefits/after-you-apply.md]: @tiffanybbrown-va a few of these lists still have the `.last` class on the last list item - that can be removed, right?

**Comment by melwoodard** [comment]
Would it make sense to merge this branch into #4098 instead of master to guarantee this all goes out together? Or is that bad form?

**Comment by jcmeloni-usds** [comment]
@rroueche ok w/ me.

**Review by jcmeloni-usds — APPROVED** [review]
LGTM after addressing/answering Mel's question.

**Comment by rroueche** [comment]
Hmm, I'm sort of nervous about merging both on the same day bc there may be a lot of conflicts  -- since we're editing the same spots on the same pages in this branch and the other branch being merged in.

My preference would be to merge #4098 on Friday, then merge master back into this branch so we capture those changes to the /education pages here, and then I'll go through and update them to have the correct classes, and then we can check them in the heroku build for this PR and merge on Monday.

BUT -- my nervousness might be due to my lack of engineering-savvy, so if someone has a better idea that's still equally safe, I'm up for it.

**Comment by jcmeloni-usds** [comment]
Whatever is the way that makes people the most comfortable is the best way, in this situation.  There are a lot of changes and areas for things to go wonky, so a clearly delineated step-wise process is totally fine.  

So: Friday, merge #4098, deploy, rebase this branch against new master, validate updates, merge and delivery on Monday.

In that case, I'm going to put a DO NOT MERGE note in the title so some chucklehead like myself doesn't just press the green button. :)

**Comment by rroueche** [comment]
haha ok thank you @jcmeloni-usds !! that is great.

**Comment by tiffanybbrown-va** [comment]
In the meantime, I can merge #4098 into this PR and get ahead of resolving conflicts. When 4098 joins master, we can rebase again without causing any problems.

File: content/pages/disability-benefits/after-you-apply.md
Comment (tiffanybbrown-va) [inline content/pages/disability-benefits/after-you-apply.md]: Yes. It can be removed, but there's no harm in leaving it in either.

**Comment by jcmeloni-usds** [comment]
What's the best next step here to get this in production in alignment w/ whatever @rroueche & team needs?

**Comment by tiffanybbrown-va** [comment]
It looks like #4098 is still under review. I need to merge it into this PR though. I merged `master` instead thinking that 4098 had already been folded into it.

**Comment by tiffanybbrown-va** [comment]
Hey @rroueche this is throwing an error because of a broken link. Should this point somewhere else?

```
Error: Link is broken: href: "/gi-bill-comparison-tool", text: "Use the GI Bill Comparison Tool", in file: education/other-educational-assistance-programs/index.html
```

**Comment by rroueche** [comment]
@tiffanybbrown-va hmm, that URL seems to be correct. Is it a react app? I remember pointing to a react app (as opposed to a static page) w/a relative link caused checks to fail in a past PR...

**Comment by rroueche** [comment]
I just resolved the conflict, hopefully that helps!

**Comment by tiffanybbrown-va** [comment]
No. No dice @rroueche. I guess the only fix is to use a full URL.

**Comment by ayaleloehr** [comment]
@tiffanybbrown-va and @rroueche I think that link was missing a trailing slash, so I tried adding one. Hope that fixes it!

If we don't want the trailing slash on there, I believe the alternative would be to add a line [here](https://github.com/department-of-veterans-affairs/vets-website/blob/7b3887e40b2b33cb45a1248134fab93ccb2338f6/script/build.js#L324) that doesn't have the trailing slash either.

**Comment by tiffanybbrown-va** [comment]
Looks like that did the trick @ayaleloehr. Thanks!

**Comment by rroueche** [comment]
Thanks for fixing the link!

#4098 is kind of on hold indefinitely now... like it might still be another 2-3 weeks until we merge it. SO, is it bad to keep waiting on that?

If it is bad, then let's go ahead and merge this to master, then once it goes live, I'll merge master back into 4098, resolve all the ed conflicts (to preserve 4098 static content changes, but accept new class names from master), and then we should be good to go all around.

**Comment by tiffanybbrown-va** [comment]
I'm pretty sure I merged #4098 into this branch, so we'll need to close this anyway. 

I'll create a new PR without those content changes so we can get this into master.

**Comment by rroueche** [comment]
@tiffanybbrown-va ok sounds good thank you!

**Comment by tiffanybbrown-va** [comment]
Closing in favor of department-of-veterans-affairs/vets-website#5379

**Pull request closed without merging**
